CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v0.5.0
+
+- New: include compiler crashes in ICE regression definition
+- New: ANSI escape code colored standard stream output
+- New: More concise final report intended for copy/paste into rust-lang/rust issue tracker threads. This eliminates some of the Markdown elements that are not generally included in threads by users
+- New: Add bisect-rustc version to the final report
+- New: Add host triple to the final report
+- New: Add command line args to reproduce the reporter's bisect-rustc tests to final report
+- Fix: end date reporting when `--end` option used without `--start` option
+- Updated: Standard stream reporting format for improved readability during execution
+- Updated: Final report instructions for regression reporting
+- Updated: 
```

Comment:
```suggestion
```

Reply:
Thanks! Sorry I missed this.  I pushed https://github.com/rust-lang/cargo-bisect-rustc/pull/87/commits/a1af249541815c5c718a45467fbfe24a7940c469 with a few edits before you submitted this.

Reply:
race-condition :)

Reply:
ha!